From 723545ebe9933b90ea58dc125e4987c6bcb04ade Mon Sep 17 00:00:00 2001
From: Anatol Pomozov <anatol.pomozov@gmail.com>
Date: Tue, 2 May 2023 13:34:24 -0700
Subject: [PATCH] Fix version in libfdt/meson.build

Currently meson generates file /usr/lib/libfdt.so.1.6.0 that does not match Makefile build version.
---
 libfdt/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libfdt/meson.build b/libfdt/meson.build
index 240bdf41..240dab9e 100644
--- a/libfdt/meson.build
+++ b/libfdt/meson.build
@@ -18,7 +18,7 @@ sources = files(
 
 libfdt = library(
   'fdt', sources,
-  version: '1.6.0',
+  version: '1.7.0',
   link_args: ['-Wl,--no-undefined', version_script],
   link_depends: 'version.lds',
   install: true,
